tv.plex.PlexDesktop.yml

```diff
@@ -0,0 +1,117 @@
+app-id: tv.plex.PlexDesktop
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: Plex
+finish-args:
+  - --socket=x11
+  - --socket=pulseaudio
+  - --device=all
+  - --share=ipc
+  - --share=network
+  - --talk-name=org.freedesktop.ScreenSaver
+cleanup:
+  - /include
+  - /bin/cjpeg
+  - /bin/djpeg
+  - /bin/jpegtran
+  - /bin/rdjpgcom
+  - /bin/tjbench
+  - /bin/wrjpgcom
+  - "*.a"
+  - "*.la"
+modules:
+  - name: plex-metadata
+    buildsystem: simple
+    build-commands:
+      - install -Dm644 tv.plex.PlexDesktop.desktop /app/share/applications/tv.plex.PlexDesktop.desktop
+      - install -Dm664 tv.plex.PlexDesktop.png /app/share/icons/hicolor/256x256/apps/tv.plex.PlexDesktop.png
+      - install -Dm664 tv.plex.PlexDesktop.metainfo.xml /app/share/metainfo/tv.plex.PlexDesktop.metainfo.xml
+    sources:
+      - type: file
+        path: tv.plex.PlexDesktop.desktop
+      - type: file
+        path: tv.plex.PlexDesktop.png
+      - type: file
+        path: tv.plex.PlexDesktop.metainfo.xml
```

Comment:
seriously all three files should be part of your upstream build. A desktop app that doesn't ship a .desktop or an icon file, it's basically a bug....

Reply:
Like I said in the other PR, we are only shipping snaps and flatpaks, and both builds add the icons and desktop files. What you consider upstream build (the tarball containing the binaries) is not a package we are handing to users. It is simply the output directory from cqtdeployer, archived. We are also not making it artificially difficult to grab that tarball, but it should be clear to anyone downloading it that it's not an official release. In fact, I added the tarball build precisely to make it easily downloadable (instead of pulling the snap file like spotify does, which is a pain).

Reply:
I will rephrase: _they should be part of the main build artifacts._ 

**Neither is specific to Flathub** (can't speak of snap, but MANY MANY packages ship and install these files and we can use them *verbatim*).

Reply:
Plenty of packages do it this way, including tv.plex.PlexHTPC that was merged just days ago (that you also reviewed). Not sure why you are insisting on changing it in this case.

Reply:
We have talked about this with Nick, and we both think that as an upstream, you decide where you want to store these files. The requirement of shipping them with upstream code/tarball is pretty much for downstream submitters.